main.go

```diff
@@ -45,10 +50,8 @@ func main() {
 			APIKey = ctx.String("api-key")
 		}
 
-		if APIKey == "" && ctx.BoolT("help") != false {
-			cli.ShowAppHelp(ctx)
-			fmt.Println("Must provide API Key via DIGITALOCEAN_API_KEY environment variable or via CLI argument.")
-			os.Exit(1)
+		if APIKey == "" && !ctx.Bool("help") && !ctx.Bool("version") {
+			return errors.New("Must provide API Key via DIGITALOCEAN_API_KEY environment variable or via CLI argument.")
```

Comment:
don't capitalize the error string, the place that prints the error will add the capitalization in a message like:

```
log.Fatalf("Invalid argument, %v", err) // if err is capitalized, the message will look weird
```